assets/locales/en/main.json

```diff
@@ -181,6 +181,12 @@
       "tos-link": "Terms of service",
       "privacy-link": "Privacy policy"
     },
+    "analytics": {
+      "export-csv": "Export to CSV",
```

Comment:
This text is different than the default message text. We should stick with "Export as CSV".